cinema/serializers.py

```diff
@@ -1 +1,131 @@
-# write serializers here
+from rest_framework import serializers
+from .models import Genre, Actor, CinemaHall, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+    def get_full_name(self, obj):
+        return f"{obj.first_name} {obj.last_name}"
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["id", "name", "rows", "seats_in_row", "capacity"]
```

Comment:
The `capacity` field is included in the `fields` list of `CinemaHallSerializer`, but it is not defined as a serializer field or method. You should add a `SerializerMethodField` for `capacity` and implement a method to calculate it, or remove it from the `fields` list if it's not needed.